malaria-app-android/src/main/java/com/peacecorps/malaria/activities/RapidFireGame.java

```diff
@@ -236,17 +241,31 @@ public RapidFireTimeCounter(long millisInFuture, long countDownInterval) {
         @Override
         public void onTick(long l) {
             timer.setText(""+l/1000);
+            millisLeft=l;
         }
 
         @Override
         public void onFinish() {
             counter.cancel();
             prepNextQues();
         }
+    }
 
-
+    @Override
+    protected void onPause() {
+        counter.cancel();
+        counter= null;
```

Comment:
Apply proper spacing between the instance and assignment operator. Here and at other places too.